packages/medusa/src/commands/build.ts

```diff
@@ -122,6 +122,7 @@ export default async function ({ directory }: BuildArgs) {
 
   if (error) {
     console.log(`Failed to load medusa-config.js`)
```

Comment:
This shouldn't be a `.log`, it should be an `.error` as well

Reply:
shouldn't we import and use the logger instead?